shor/gates.py

```diff
@@ -507,6 +1460,24 @@ def to_matrix(self) -> np.ndarray:
 
 
 class Init_y(_Gate):
+    """
+    Initialize the qubits along y axis or |-> basis
+    Inherits from gate class
+    Currently not implimented in qiskit and can only be used on shor simulator
```

Comment:
We can actually take advantage of the "to_gates" method here.
Translating this to a series of gates that do work on ibmq.

Reply:
nit. implemented

Reply:
Thats a good idea, maybe after we launch we can add this capability.